### DISCUSSION TITLE
CLI Commands like Close Ongoing Sessions?

### DISCUSSION THREAD

**Discussion by Xephyren** [ClassicOldSong/Apollo#101] [Q&A] [answered]
Are there any commands or hotkeys for exit/close session or restart we can pass to Apollo via CLI?  I looked through sunshine's documentation and --help entry via CLI, but can't seem to figure it out.

I know Quit Session on Artemis will close the app and run undo scripts (including resetting Virtual Desktop settings), but sometimes I want to continue work on the server before closing everything, so I'll back out on Artemis without quitting.  Once I'm finished on the server, the primary display is set to the virtual desktop, so I can't access the notification tray to Force Close on Apollo.  Currently, going back through Artemis to quit the session works, but I figure there has to be a better way to do it when I'm already on the server.

**Comment by ClassicOldSong** [comment, marked as answer]
You can quit the session from web ui, with any device that can access the web ui.

There's no other ways to do this now and the security model for controlling through CLI is unclear.
